res/schema.xml

```diff
@@ -582,4 +582,31 @@ reapplying those migrations.
       UPDATE library SET filetype='aiff' WHERE filetype='aif';
     </sql>
   </revision>
+  <revision version="40" min_compatible="3">
+    <description>
+      Suggestion tables added.
+    </description>
+    <sql>
+      CREATE TABLE IF NOT EXISTS suggestion_library (
+        id INTEGER PRIMARY KEY AUTOINCREMENT,
+        artist varchar(48),
+        title varchar(48),
+        playcount INTEGER,
+        match FLOAT,
+        duration FLOAT,
+        location varchar(512));
+
+       CREATE TABLE IF NOT EXISTS suggestion_playlists (
```

Comment:
Not sure how the name is generated.
Do we have a use case for an individual playlist name? 

Maybe it is a good idea to keep this table for future suggestion playlist metadata.

I think we have always a reference track ID.
This needs to be added.

Reply:
Actually the ID and the Name is both saved with the trackId for now, I have used the (Track Title and Artist) name to save and look up playlists but that didn't work, so I tried to reference the both track_id as a POC.

That shows that, a new external playlist table model is needed such as suggestionPlaylistModel.